src/test/java/io/cdap/plugin/sfmc/connector/MarketingConnectorTest.java

```diff
@@ -116,13 +132,12 @@ public void testConvertRecord() throws ETSdkException, NoSuchFieldException {
     Mockito.when(MarketingCloudClient.create("clientId", "clientSecret", "authEndpoint",
             "soapEndpoint")).thenReturn(cloudClient);
     Mockito.when(cloudClient.fetchObjectSchema(SourceObject.MAILING_LIST)).thenReturn(cloudObjectInfo);
-    marketingConnectorConfig1.convertRecord(cloudObjectInfo, builder1, object);
+    cloudConversion.convertRecord(cloudObjectInfo, builder1, object);
   }
```

Comment:
Add the assertions, what you are testing here.

Reply:
Done